source/geometry/IntVector2D.ooc

```diff
@@ -23,7 +23,7 @@ IntVector2D: cover {
 	x, y: Int
 	area ::= this x * this y
 	square ::= this x == this y
-	empty ::= !(this x > 0 && this y > 0)
+	isZeroVector ::= this size x <= 0.0f || this size y <= 0.0f
```

Comment:
Nowhere in the current code at least. Except for boxes, but they should have their own properties.